release.mk

```diff
@@ -115,30 +115,57 @@ endef
 #
 .PHONY: minor-release
 minor-release:
+	@echo "INFO: Create GitHub label backport for the version $(RELEASE_VERSION)"
+	$(MAKE) create-github-label NAME=backport-$(RELEASE_BRANCH)
```

Comment:
Even though the GH label creation happens somewhere else, let's force the backport to be created just in case. This will be later on used for the `docs` PR to be backported from `main` to `8.x` and the new minor branch.